modules/ipi-install-bmc-addressing.adoc

```diff
@@ -8,6 +8,8 @@
 
 Most vendors support Baseboard Management Controller (BMC) addressing with the Intelligent Platform Management Interface (IPMI). IPMI does not encrypt communications. It is suitable for use within a data center over a secured or dedicated management network. Check with your vendor to see if they support Redfish network boot. Redfish delivers simple and secure management for converged, hybrid IT and the Software Defined Data Center (SDDC). Redfish is human readable and machine capable, and leverages common internet and web services standards to expose information directly to the modern tool chain. If your hardware does not support Redfish network boot, use IPMI.
 
+If you need to modify the BMC address or credentials after installation, see the _Additional resources_ section.
```

Comment:
This epic applies to BMC address only, not credentials. While your statement is correct in the context of this doc, the following must be made clear about editing the BMC address specifically:

The BMC address can be edited in two ways/cases: with the detached annotation, and when the BMH is still in the 'Registering' state (in BMO). 
[From my observation, a BMH very quickly moves out of the Registering state as soon as the credentials are verified, but if you enter the BMC address incorrectly (by mistake, for example), the BMH will not move past the Registering state.] While the BMH is in the Registering state, the BMC address can be edited in the BMH resource (without the need for detachment). In all other BMH states, you have to use the detach annotation.

Reply:
@MahnoorAsghar I've made some changes. Can you have a look and give an LGTM if it's okay?